web/models/project.ex

```diff
@@ -25,6 +25,8 @@ defmodule CodeCorps.Project do
 
     has_many :project_categories, CodeCorps.ProjectCategory
     has_many :categories, through: [:project_categories, :category]
+    has_many :project_skills, CodeCorps.ProjectSkill
+    has_many :skills, through: [:project_skills, :skill]
```

Comment:
I may suggest separating this `has_many` two-fer group to make it a little easier on the eyes.